backend/src/repositories/ticker_covariance.py

```diff
@@ -1,16 +1,29 @@
 """ Defines the Ticker Covariance repository """
-
+import numpy as np
 from models import TickerCovariance
 
 
 class TickerCovarianceRepository:
+    @staticmethod
+    def get_covariance_matrix(symbols):
+        length = len(symbols)
```

Comment:
Maybe more apt to use d so shape will be a dxd matrix

Reply:
changed to dim